includes/compatibility/class-wcpdf-compatibility-third-party-plugins.php

```diff
@@ -19,10 +19,10 @@ class Third_Party_Plugins {
 	function __construct()	{
 		// WooCommerce Subscriptions compatibility
 		if ( class_exists('WC_Subscriptions') ) {
-			if ( version_compare( \WC_Subscriptions::$version, '2.0', '<' ) ) {
+			if ( version_compare( \WC_Subscriptions::$version, '2.0.17', '<' ) ) {
```

Comment:
Is there a reason you upped this version? AFAIK the deprecation of `woocommerce_subscriptions_renewal_order_created` was done at 2.0.0 already (not that I expect many people to still use such an old version...)